BUILD.gn

```diff
@@ -59,14 +59,24 @@ config("branding") {
 # The one in $target_gen_dir is used for the actual TSC build later one
 # and the one in //electron/electron.d.ts is used by your IDE (vscode)
 # for typescript prompting
-npm_action("build_electron_definitions") {
-  script = "gn-typescript-definitions"
-  args = [ rebase_path("$target_gen_dir/tsc/typings/electron.d.ts") ]
-  inputs = auto_filenames.api_docs + [ "package-lock.json" ]
-
-  outputs = [
-    "$target_gen_dir/tsc/typings/electron.d.ts",
-  ]
+if (is_win) {
+  npm_action("build_electron_definitions") {
+    script = "gn-typescript-definitions-win"
```

Comment:
Only the `script = ""` needs to be wrapped in the `if (is_win)` (It's the only thing that needs to change, no need to duplicate the entire block)